content/en/blog/_posts/2020-08-21-Moving-Forward-From-Beta/index.md

```diff
@@ -0,0 +1,111 @@
+availability (GA), also known as "stable". GA features are part of Kubernetes, with a
+commitment that they are staying in place throughout the current major version.
+
+Having beta features on by default lets Kubernetes and its contributors get valuable
+real-world feedback. However, there's a mismatch of incentives. Once a feature is enabled
+by default, people will use it. Even if there might be a few details to shake out,
+the way Kubernetes' REST APIs and conventions work mean that any future stable API is going
+to be compatible with the most recent beta API: your API objects won't stop working when
+a beta feature graduates to GA.
+
+For the API and its resources in particular, there's a much less strong incentive to move
+features from beta to GA than from alpha to beta. Vendors who want a particular feature
+have had good reason to help get code to the point where features are enabled by default,
+and beyond that the journey has been less clear.
+
+KEPs track more than code improvements. Essentially, anything that would need
+communicating to the wider community merits a KEP. That said, most KEPs cover
+Kubernetes features (and the code to implement them).
+
+You might know that [Ingress](/docs/concepts/services-networking/ingress/)
+has been in Kubernetes for a while, but did you realize that it actually went beta in 2015? To help
+drive things forward, Kubernetes' Architecture Special Interest Group (SIG) have a new approach in
+mind.
+
+## Avoiding permanent beta
+
+For Kubernetes REST APIs, when a new feature's API reaches beta, that starts a countdown.
+The beta-quality API now has **nine calendar months** to either:
+- reach GA, and deprecate the beta, or
+- have a new beta version (_and deprecate the previous beta_).
+
+To be clear, at this point **only REST APIs are affected**. For example, _APIListChunking_ is
+a beta feature but isn't itself a REST API. Right now there are no plans to automatically
+deprecate _APIListChunking_ nor any other features that aren't REST APIs.
+
+If a REST API reaches the end of that 9 month countdown, then the next Kubernetes release
+will deprecate that API version. There's no option for the REST API to stay at the same
+beta version beyond the first Kubernetes release to come out after the 9 month window.
+
+### What this means for you
+
+If you're using Kubernetes, there's a good chance that you're using a beta feature. Like
+I said, there are lots of them about.
+As well as Ingress, you might be using [CronJob](/docs/concepts/workloads/controllers/cron-jobs/),
+or [PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/), or others.
+There's an even bigger chance that you're running on a control plane with at least one beta
+feature enabled.
+
+If you're using or generating Kubernetes manifests that use beta APIs like Ingress, you'll
+need to plan revise those. The current APIs are going to be deprecated following a
```

Comment:
missing word:
you'll need to plan to ...

Reply:
The (`You` and `contributors`) section headings seem to overlap or could be combined.
Could clarify by something like, `The motivation for contributors is clear: get features stable.` ...
The second sentence in `What this means for Kubernetes contributors` runs on and on.